identify/extensions.py

```diff
@@ -269,6 +269,7 @@
     '.mention-bot': EXTENSIONS['json'] | {'mention-bot'},
     '.npmignore': {'text', 'npmignore'},
     '.pdbrc': EXTENSIONS['py'] | {'pdbrc'},
+    '.prettierignore': {'text', 'gitignore'},
```

Comment:
this should also mention `prettierignore` specifically in case one wants to target that as well